WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2167,7 +2167,52 @@ static GSourceFuncs _handlerIntervention =
                 }
                 g_mkdir_with_parents(wpeDiskCachePath, 0700);
 
+#if WEBKIT_CHECK_VERSION(2, 38, 0)
+                uint32_t webProcessLimit{ 0 };
+                uint32_t networkProcessLimit{ 0 };
+
+                if (_config.MemoryPressure.Value().empty() == false) {
```

Comment:
I think it will be better to adjust the config file itself. Let's split up the memorypressure config line as it is no longer directly going into an environment variable and make it all seperate values in the config file. That makes filling the values easier as well as handling in the code as there is no need to parse the string. It is not a problem if the config file is not backwards compatible, we just make that conditional with the same flag as used in the code (see my comment about that).